src/styles/_colors.css

```
@@ -0,0 +1,7 @@
+:root {
+  --color-background: #fbf2f2;
+  --color-icons: #d84064;
+  --color-text: #d84064;
+  --color-items-background: #f4cbad;
+  -color-single-card: #c05757;
```

This variable is missing a `-`

```suggestion
  --color-single-card: #c05757;
```

---

Good catch ! Thanks :)